CHANGELOG.md

```diff
@@ -6,28 +6,32 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+## [1.7.0] - 2020-05-29
+
 ### Fixed
-- Title of status page is now `Conjur Status` again, rather than only
-  `Conjur` ([conjurinc/dap-support](https://github.com/conjurinc/dap-support/issues/75)).
-- Policy load API endpoints now default to the `application/x-yaml` content-type
-  if no content type is provided in the request
-  ([conjurinc/dap-support#74](https://github.com/conjurinc/dap-support/issues/74)).
 - The k8s authenticator correctly authenticates an app using the host ID to specify
   the k8s resource constraints and an annotation to specify the authenticator
-  container name using the "authn-k8s" prefix
-  ([cyberark/conjur#1535](https://github.com/cyberark/conjur/issues/1535),
-  [conjurinc/dap-support#79](https://github.com/conjurinc/dap-support/issues/79)).
+  container name using the "authn-k8s" prefix ([cyberark/conjur#1535](https://github.com/cyberark/conjur/issues/1535), [conjurinc/dap-support#79](https://github.com/conjurinc/dap-support/issues/79)).
+- Fixed exception in `conjurctl` when loading policy ([conjurinc/dap-support#80](https://github.com/conjurinc/dap-support/issues/80)).
 
 ### Changed
-- Change ActiveSupport to use sha1 instead of md5
+- Updated the title of status page to `Conjur Status` from `Conjur` ([conjurinc/dap-support](https://github.com/conjurinc/dap-support/issues/75).
+- Policy load API endpoints now default to the `application/x-yaml` content-type if no content type is provided in the request ([conjurinc/dap-support#74](https://github.com/conjurinc/dap-support/issues/74)).
+- Change ActiveSupport to use SHA1 instead of MD5 ([cyberark/conjur#1418](https://github.com/cyberark/conjur/issues/1418)).
```

Comment:
fwiw, the TWs have said that starting changelog lines under "Changed" with "Change(d)" stutters in the release notes - this could probably be changed to "ActiveSupport uses"